kayenta-core/src/main/java/com/netflix/kayenta/metrics/SynchronousQueryProcessor.java

```diff
@@ -146,28 +164,15 @@ private long getBackoffPeriodMs(int retryAttemptNumber) {
         * retryConfiguration.getBackoffPeriodMultiplierMs();
   }
 
-  private boolean isRetryable(RetrofitError e) {
-    if (isNetworkError(e)) {
-      // retry in case of network errors
-      return true;
-    }
-    if (e.getResponse() == null) {
-      // We don't have a network error, but the response is null. It's better to not retry these.
-      return false;
-    }
-    HttpStatus responseStatus = HttpStatus.resolve(e.getResponse().getStatus());
+  private boolean isRetryable(SpinnakerHttpException e) {
```

Comment:
Wouldn't it be simpler to do
```suggestion
  private boolean isRetryable(SpinnakerServerException e) {
```
and then keep an isNetworkError that also took a SpinnakerServerException argument?

Reply:
yeah, I think I see a way to do that. I'll update my branch